lib/pxenforcer.js

```diff
@@ -261,6 +261,11 @@ class PxEnforcer {
             pxApi.evalByServerCall(ctx, this._config, (action) => {
                 ctx.riskRtt = Date.now() - startRiskRtt;
 
+                if (this.config.config.REMOTE_CONFIG_ENABLED && ctx.remoteConfigLatestVersion > this._config.REMOTE_CONFIG_VERSION) {
```

Comment:
This is much better but the `isRemoteConfigOutdated` is not necessary it actually creates additional overhead because we need to make sure to set it back to false after we're updating the the Enforcer.
This is currently not happening in the BDExpress PR but we can avoid it by just comparing the version over there and removing the `ctx.remoteConfigLatestVersion > this._config.REMOTE_CONFIG_VERSION` from the `if` statment